CONFLICT.md

```diff
@@ -1,41 +1,44 @@
-# Article I
-## Purpose
+# Conflict of Interest Policy
+
+## Article I
+
+### Purpose
 
 The purpose of the conflict of interest policy is to protect this tax-exempt organization's (Organization) interest when it is contemplating entering into a transaction or arrangement that might benefit the private interest of an officer or director of the Organization or might result in a possible excess benefit transaction. This policy is intended to supplement but not replace any applicable state and federal laws governing conflict of interest applicable to nonprofit and charitable organizations.
 
+## Article II
 
-# Article II
-## Definitions
+### Definitions
 
-### 1. Interested Person
+#### 1. Interested Person
 
-Any director, principal officer, or member of a committee with governing board delegated powers, who has a direct or indirect financial interest, as defined below, is an interested person. 
+Any director, principal officer, or member of a committee with governing board delegated powers, who has a direct or indirect financial interest, as defined below, is an interested person.
```

Comment:
This definition should be expanded to include "appointed positions"

Reply:
Adjusted in latest commit